docs/usage/configuration-options.md

```diff
@@ -1918,6 +1918,8 @@ The `postUpgradeTasks` configuration consists of three fields:
 
 A list of commands that are executed after Renovate has updated a dependency but before the commit is made.
 
+You can use variable templating in your commands if [`allowPostUpgradeCommandTemplating`](https://docs.renovatebot.com/self-hosted-configuration/#allowpostupgradecommandtemplating) is enabled.
```

Comment:
@HonkingGoose shouldn't we use a relative link?

Reply:
I've checked the rest of the documentation and it seems to use absolute links everywhere

Reply:
> @HonkingGoose shouldn't we use a relative link?

Probably, see [MkDocs manual, linking to pages](https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages).

> I've checked the rest of the documentation and it seems to use absolute links everywhere

You're right, but we probably should try to avoid that. I know I use a lot of links with the full path and domain, like: `https://docs.renovatebot.com/self-hosted-configuration/#allowpostupgradecommandtemplating` but that's prone to break if we ever want to change our domain.

The problem for me is that it's hard to see if the relative link work properly when you're browsing in GitHub _and_ in the published docs. I think you can do some thing with `git submodules` to pull in your changes into the docs site repository and preview if it works there somehow. But this is not well documented.

Maybe we should make a document that explains how I can preview links properly @viceice? I think you have some complicated way to check it?